lib/Transformer.js

```diff
@@ -243,6 +243,13 @@ export class Transformer {
           contextUrl = context;
           ({'@context': ctx} = await this._getDocument({url: contextUrl}));
         }
+
+        // support parsing context lists recursively
+        if(Array.isArray(ctx)) {
+          await this._updateContextStack({contextStack, contexts: ctx});
```

Comment:
Do we need to check to make sure we're not encountering a resolution loop here or is that being done elsewhere? Thinking about the case where the [context file references another context file](https://github.com/w3c-ccg/security-vocab/blob/273bd5ef4e1c3e84968fe66add00a7bdd0a25c11/contexts/security-v2.jsonld#L4) and that reference points back to the original context which would create a loop.

Reply:
Thanks @kdenhartog, I've pushed the changes ([commit](https://github.com/mattrglobal/digitalbazaar-cborld/pull/2/commits/51691cf130b38fa29ef06f92472b26d72f645829)) that handles circular context resolution. Do you think we should raise an error in this case or skip processing already visited contexts? Thanks!

Reply:
If there's a way to skip processing and break out of the loop that would be ideal. If not I think raising an error works just as well. Might be worth checking the JSON-LD lib to see how they handle it as well.

Reply:
Thanks @kdenhartog, it's a very good point. The `jsonld` lib will raise an error when cyclical context had detected (https://github.com/digitalbazaar/jsonld.js/blob/master/lib/ContextResolver.js#L145-L157), I will update the code to match this behavior.  🙏

Reply:
Updated 😎